hr_timesheet_sheet_attendance/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "HR Timesheet Sheet Attendance",
+    "version": "15.0.1.0.0",
+    "category": "Human Resources",
+    "sequence": 80,
+    "license": "AGPL-3",
+    "author": "BizzAppDev, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/timesheet",
+    "depends": ["hr_attendance", "hr_timesheet_sheet"],
+    "external_dependencies": {"python": ["freezegun"]},
```

Comment:
Isn't `freezegun` is already in core Odoo?

Reply:
Yes are right I thought it is required to install